cypress/integration/homepage-smoke.spec.js

```diff
@@ -185,8 +185,7 @@ describe(`Home page tests : Tests execution date and time : ${new Date()}`, () =
 	});
 
 	it('It matches the event date, time and location with previous page', () => {
-		var eventDate;
-		var eventTime;
+		var eventDateAndTime;		
 		homePage.getFindEventLink().click();
 		cy.get('#events_search_month').select('November 2020');
```

Comment:
Hardcoding dates like this will cause the specs to fail next month; can we make the value dynamic instead?

Reply:
Resolved